internal/voting/voting.go

```diff
@@ -41,13 +41,36 @@ func initializeVoteStore(ctx context.Context, db sql.TxMaker) error {
 	}
 	defer tx.Rollback(ctx)
 
+	compiledResolutions := resolutions.ListResolutions()
+	resMap := make(map[string]bool, len(compiledResolutions))
+	for _, name := range compiledResolutions {
+		resMap[name] = true
+	}
+
+	dbResolutions, err := getResolutionTypes(ctx, tx)
+	if err != nil {
+		return fmt.Errorf("failed to get resolution types: %w", err)
+	}
+
+	if len(dbResolutions) != 0 {
```

Comment:
Why do you check if `len(dbResolutions) != 0`?. Seems unnecessary

Reply:
Ah, sorry, relic from the past commits. removed it